EIPS/eip-3712.md

```diff
@@ -0,0 +1,340 @@
+---
+eip: 3712
+title: Multiple Fungible Token Standard
+author: Zhao Li(@1999321)、Yuefei Tan (@whtyfhas)、Derek Zhou (@zhous)  
+discussions-to: https://ethereum-magicians.org/t/eip-3712-standard-for-multiple-types-of-fungible-tokens/6912
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-08-01
+---
+
+## Simple Summary
```

Comment:
We recently deprecated the "Simple Summary" in favor of a description field in the preamble. Please see the [EIP template ](https://github.com/ethereum/EIPs/blob/master/eip-template.md)for an example.